exec/java-exec/src/main/java/org/apache/drill/exec/server/options/TypeValidators.java

```diff
@@ -139,6 +140,10 @@ public BooleanValidator(String name, boolean def) {
     public BooleanValidator(String name, boolean def, boolean isAdminOption) {
       super(name, Kind.BOOLEAN, OptionValue.createBoolean(OptionType.SYSTEM, name, def), isAdminOption);
     }
+
+    public OptionValue loadConfigDefault(DrillConfig bootConfig, String name, String configPath){
+      return OptionValue.createBoolean(OptionType.SYSTEM, name, bootConfig.getBoolean(configPath+name));
```

Comment:
Instead of `configPath+name` multiple times, if we turn `configPath` into a constant, and get `name` from the validator itself, we want:

```
...getBoolean(getConfigProperty());
...
private String getConfigProperty() {
    return CONFIG_PATH + getOptionName(); }
```